.github/workflows/buildtest.yml

```diff
@@ -119,7 +122,10 @@ jobs:
         # Build & TEST & DEPLOY STORM
         #####
       - name: Git clone
-        run: sudo docker exec storm git clone --branch $STORM_BRANCH $STORM_GIT_URL /opt/storm
+        run: |
+        # git clone cannot clone individual commits based on a sha and some other refs
+        # this workaround fixes this and fetches only one commit
+          sudo docker exec storm bash -c "mkdir /opt/storm; cd /opt/storm; git init && git remote add origin ${STORM_GIT_URL} && git fetch --depth 1 origin ${STORM_BRANCH} && git checkout FETCH_HEAD"
```

Comment:
We should clone the complete repo here.
The versioning uses git tags which are missing if only one commit is pulled. While this still works in Storm without problems, stormpy will encounter issues with determining the right version. As we are also deploying this part to Docker, it should contain the complete repo.

Reply:
If have fixed it so that only when we deploy will the whole history be cloned.